nncf/common/graph/graph.py

```diff
@@ -148,17 +148,23 @@ def __init__(
         self.parallel_input_port_ids = parallel_input_port_ids
```

Comment:
What do you think using MultiDiGraph can resolve the issue with `parallel_input_port_ids` https://networkx.org/documentation/stable/reference/classes/multidigraph.html?

Reply:
@vshampor, could you please share a ticket number for this issue?

Reply:
I could find the 110944 to that accord.